vacancies/adidas-developer.html

```diff
@@ -9,7 +9,7 @@
 
 		<meta property="og:url" content="https://code-orange.nl/vacancies/adidas-developer.html">
 		<meta property="og:title" content="adidas Developer @ Code Orange">
-		<meta property="og:image" content="https://code-orange.nl/img/square.png">
+		<meta property="og:image" content="https://code-orange.nl/img/logo.svg">
```

Comment:
I don't think SVG content is supported for `og:image`, images are supposed to be a certain fixed size as well.